content/blog/2020-04-06-april-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,160 @@
+---
+title: April '20 DVC❤️Heartbeat
+date: 2020-04-06
+description: |
+  DVC discussions around the web,
```

Comment:
can we make them more or less useful? like summary of things inside?

Reply:
yeah i changed it to:

 `Catch up on new DVC releases, talks, and projects in our community. 
  This month, learn what we're up to in MLOps, CI/CD, and the 
  intersection of data science and software engineering.`

suggestions welcome